### ISSUE TITLE
Dual subtitles

### ISSUE THREAD

**Issue by Seby68** [FrostCo/AdvancedProfanityFilter#492]
Hi Richard, I found an extension, which uses dual subtitles in Netflix. With such app we could have much more chance to stop profanity. I noticed that with the extension in use, audio #1 is checked, while audio #2 is not. The extension is NflxMultiSubs 2021 (Netflix Multi. Subtitles). You can take a look at it, I think it's worth it .I attach two files to give you an idea.
![Screenshot (156)](https://user-images.githubusercontent.com/110528035/226668914-463a738c-b4de-4a07-ae0b-81a722136f4d.png)
![Screenshot (157)](https://user-images.githubusercontent.com/110528035/226668951-68b0eb1a-f63a-47cd-b9bb-6dd952ed7c3f.png)

**Comment by Seby68** [comment]
Hi Richard, how can you, check out this extension, as far as we are. Evaluate, which could be very useful, because we could filter two subtitle languages, at the same time. For example, I have included English, Italian, Polish and Spanish in your extension. Hello and thanks in advance.

**Comment by richardfrost** [comment]
@Seby68 I took a look at it, and I think I should be able to support it with a little bit of work. I'll let you know how it goes in the next few days.

**Comment by Seby68** [comment]
Thank you very much . I hope you succeed, there may be, quite useful. Good work.
—
—

**Comment by richardfrost** [comment]
@Seby68, here is a preview version that should support what you are looking for. If this works alright for you I'll go ahead and release a version with these new code changes.

[extension-mv3-chrome-netflix-dual-subs.zip](https://github.com/FrostCo/AdvancedProfanityFilter/files/11182749/extension-mv3-chrome-netflix-dual-subs.zip)

_Note: Learn more about using an [unpacked extension here](https://github.com/FrostCo/AdvancedProfanityFilter/wiki/Load-Unpacked-Extension)._

I wasn't able to find a good way to keep a single config working in both cases where "NflxMultiSubs 2021" was installed and not installed. As such, I won't be making a change to the official Netflix config included in the extension. So, if someone wanted to use Netflix normally (without "NflxMultiSubs 2021", or if they only care about muting for the primary language subtitle) they won't have to do anything.

If you want to enable muting for both the primary and secondary subtitles (available in NflxMultiSubs 2021) then you'll need to add this to your Custom Sites config, which will take precedence over the included Netflix config:

```json
{
  "www.netflix.com": [
    {
      "mode": "watcher",
      "subtitleSelector": "div.player-timedtext-text-container span span &&& div.nflxmultisubs-subtitle-container svg.nflxmultisubs-subtitle-svg text tspan"
    }
  ]
}
```
_Note: If you have other "Custom Sites config" already, feel free to send it to me and I can help you combine it with this new one._

Let me know how it works for you, and if it is acceptable I'll get an official release ready.

**Comment by Seby68** [comment]
[Thanks, I'll let you know by testing it well. Have a nice weekend.

**Comment by Seby68** [comment]
Hi Richard, already tested. It works worse than before, it doesn't even stop, not even the text in the second title, while before it didn't stop only the audio. Ok I look forward to new developments. Thank you.

**Comment by Seby68** [comment]
Sorry Richard, but I didn't install the zip file, I'll let you know.

**Comment by Seby68** [comment]
Richard, I tested one language 4 times and the other 2 times and it worked fine. Streaming over audio also stops fine. By my choice I lengthened the downtime, by 1200, because sometimes the second audio arrives late on the subtitles. Thanks again .

**Comment by richardfrost** [comment]
Thanks for testing that @Seby68. I'll go ahead and get those changes added into the next official version so you can start using it then. I will try to get it submitted in the next day or so, and then it will just depend on how long the various store's review process takes before you'll be able to download the update. I'll update here once its available.

**Comment by richardfrost** [comment]
This has been included in version [3.1.0](https://github.com/richardfrost/AdvancedProfanityFilter/releases/tag/3.1.0). Feel free to comment here if you have any feedback for this update.

@Seby68 you'll need to keep that custom config if you want to use the "dual subtitles" on Netflix as it isn't being included with the official supported sites.
